cmake/generic.cmake

```diff
@@ -316,5 +316,5 @@ endfunction(go_test)
 # go_extern(target_name extern_source)
 # go_extern(go_redis github.com/hoisie/redis)
 function(go_extern TARGET_NAME)
-  add_custom_target(${TARGET_NAME} env GOPATH=${GOPATH} ${CMAKE_Go_COMPILER} get ${ARGN})
+  add_custom_target(${TARGET_NAME} env GOPATH=${GOPATH} ${CMAKE_Go_COMPILER} get -d ${ARGN})
```

Comment:
Should we add `-u` also when doing `go get` so we can use the latest version of external dependencies?